guides/rails_guides/markdown/renderer.rb

```diff
@@ -160,6 +176,21 @@ def api_link(url)
             url.sub(/(?<=\.org)/, "/#{version}")
           end
         end
+
+        # Parses "ruby#3,5-6,10" into ["ruby", [3,5,6,10]] for highlighting line numbers in code blocks
```

Comment:
I'm curious about the syntax here. Is it something you came up with, or got from somewhere?

The CommonMark spec [does mention "info strings"](https://spec.commonmark.org/0.30/#example-143) (I'm not 100% sure about their purpose). They look like metadata, which could be handy here.

Reply:
Thanks for sharing, hadn't seen this. I think using `ruby highlight=3,5-6,10` might be a better format like examples in the spec. Since they suggest the first word as the language, separating with a space instead of hash would fit better. Will refactor.

Reply:
Actually, seems like Redcarpet only passes the first word. I remember hitting that previously. Redcarpet's a C extension so I don't think that's gonna be an easy one.